pagopa/io-app
# feat: [IOBP-235]  Addition of  idpay code display screen

## Short description
addition of said screen

<img width="250" alt="Screenshot 2023-09-19 at 15 40 09" src="https://github.com/pagopa/io-app/assets/60693085/234817b0-ae30-4a9d-a14d-b1f24e4e2d5b">

## List of changes proposed in this pull request
- addition of said screen
- this pr is in early-ish development and has several blockers.
- the banner has the wrong pictogram, will be updated in this PR once the new DS version is up in npm

## How to test
if the dependency is not merged, you'll need to merge the branch locally yourself.
with idpay test flag active, navigate to profile>idpay code playground>code display screen and make sure everything renders right
